Makefile

```diff
@@ -183,12 +192,13 @@ is-semantic-version:
 	[[ $(VERSION) =~ $(PATTERN) ]] || \
 	(printf '\nerror: VERSION does not match the format required by operator-sdk.\n\n' && exit 1)
 
-# Generate kustomization.yaml for bundle
-KUSTOMIZE := $(CURDIR)/bin/kustomize
-$(KUSTOMIZE): vendor/modules.txt
+# TODO: a workaround until this issue will be fixed https://github.com/kubernetes-sigs/kustomize/issues/4008
+$(KUSTOMIZE):
```

Comment:
Doesn't this introduce a type of supply-chain vulnerability like @skitt fixed in f71061f5fec92b1308e7005285710c5cfc30b006? We might need to do those verifications here.

Reply:
I did not find the checksum on the source repo. I will try to look elsewhere.

Reply:
This might be something that's better documented in an Issue for now and fixed here (and elsewhere?) next sprint.

Reply:
Fixed, I found the `checksum.txt` file on github. Thanks for the notice @dfarrell07